.gitignore

```diff
@@ -8,3 +8,5 @@ composer.phar
 .phpunit.result.cache
 .php_cs.cache
 .phpcs-cache
+config/application.config.php
+ci/20*/
```

Comment:
What is this line for?

Reply:
Running test local generate migration like `ci/2020/Version20211102200030.php` (current date)